lib/acts-as-taggable-array-on/taggable.rb

```diff
@@ -35,6 +45,21 @@ def acts_as_taggable_array_on(tag_name, *)
         end
       end
       alias_method :taggable_array, :acts_as_taggable_array_on
+
+      def define_allowed_validations!(tag_name, allowed)
```

Comment:
Not sure if naming of attributed "allowed" make a lot of sense here. "allowed:" is tag property or "property_name" for short? But i'm not 100% confident that it's a right name though.